pybamm/meshes/meshes.py

```diff
@@ -47,6 +47,9 @@ def __init__(self, geometry, submesh_types, submesh_pts):
                 submesh_types[domain](geometry[domain]["primary"], submesh_pts[domain])
             ] * repeats
 
+        # add ghost meshes
+        self.add_ghost_meshes()
```

Comment:
Does this replace the add_ghost_meshes() in discretisation.__init__() ?

Reply:
it should do hopefully, I think this is a more sensible place for it

Reply:
Yeah agree. We should remove the one in discretisation.__init__(), I think its still there. Otherwise, we will have "negative electrode_left_ghost_left_ghost" kicking around.